### ISSUE TITLE
Editorialize the Want-Content and Want-Content-Digest field sections

### ISSUE THREAD

**Issue by LPardue** [httpwg/http-extensions#1661]
These use the same mechanism, so consolidate their description. In parallel, we can also also tighten the text about qvalue use.

**Comment by jricher** [comment]
Perhaps this needs its own issue and discussion, but why are these fields using "want-" and not "accept-", which seems to be the common prefix used for other fields?

**Comment by LPardue** [comment]
To address @jricher's point: this doc is an update to RFC 3230, so we're constrained by the path already set out to us. `Want-Digest` is already a thing (although arguably of questionable value). `Want-Content-Digest` follows suit, meaning its straightforward to make changes like #1662. Picking a different prefix might be purely better but we're kind of adding new lanes to cowpaths here.

Since this issue got closed via the PR, if there's somthing you'd like to follow up on, please open a new issue.
